envhelp/makeconfig

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
```

Comment:
Missing default header. I'll add it there and in the `envhelp/pythonX` wrapper scripts.